lib/logstorage: add `coalesce` pipe, update documentation (#904)
VictoriaMetrics/VictoriaLogs: docs/victorialogs/CHANGELOG.md

## tip

> * FEATURE: [LogsQL](https://docs.victoriametrics.com/victorialogs/logsql/): add [`coalesce` pipe](https://docs.victoriametrics.com/victorialogs/logsql/#coalesce-pipe) for returning the first non-empty value from the specified list of fields, or a default value if all are empty. See [#690](https://github.com/VictoriaMetrics/VictoriaLogs/issues/690)

Reviewer: ```suggestion
* FEATURE: [LogsQL](https://docs.victoriametrics.com/victorialogs/logsql/): add [`coalesce` pipe](https://docs.victoriametrics.com/victorialogs/logsql/#coalesce-pipe) for returning the first non-empty value from the given fields. It can also return a default value when all the fields are empty. See [#690](https://github.com/VictoriaMetrics/VictoriaLogs/issues/690).
```

Author: Aslo fixed, thank you.
